src/main/java/uk/gov/digital/ho/hocs/client/casework/CaseworkClient.java

```diff
@@ -63,4 +60,9 @@ public void updateCase(UUID caseUUID, UUID stageUUID, Map<String, String> data)
         log.info("Set Case Data for Case {}", caseUUID);
     }
 
+    public CreateCaseworkCaseResponse migrateCase(CreateMigrationCaseRequest request) {
```

Comment:
I wonder if we should move this to a migration package like we've done in hocs-casework?